README.md

```diff
@@ -30,7 +30,12 @@
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add your own name logo created by you at the top of the readme file to make your readme attractive and present you as a brand in the software industry 👍 to catch the recruiters' eyes 😊